What's new page for 5.13 (#228)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/whats-new.adoc

= What's New in Hazelcast Platform Operator

> :description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.0.

Reviewer: ```suggestion
:description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.
```
I'd just using x.y here as we will add to this page if e.g. 13.1 adds new features

{description}

> NOTE: For new and improved features in Hazelcast Platform 5.5, see xref:{page-latest-supported-hazelcast}@hazelcast:ROOT:whats-new.adoc[What's new in Hazelcast Platform 5.5].

Reviewer: ```suggestion
NOTE: For new and improved features in Hazelcast Platform, see xref:{page-latest-supported-hazelcast}@hazelcast:ROOT:whats-new.adoc[What's new in Hazelcast Platform 5.5].
```

== Community Edition and Repository Changes

> In March 2024, we made some updates to the Community Edition of the Hazelcast Platform, including making the Hazelcast Platform Operator 5.13 closed-source and its cluster creation capabilities will no longer be available in the Community Edition. 

Reviewer: ```suggestion
In March 2024, we made some updates to the Community Edition of the Hazelcast Platform, including making the Hazelcast Platform Operator 5.13 closed source and its cluster creation capabilities no longer available in the Community Edition. 
```

> In March 2024, we made some updates to the Community Edition of the Hazelcast Platform, including making the Platform Operator 5.13 closed-source and its cluster creation capabilities will no longer be available in the Community Edition. 
> 
> You can find more information here: https://hazelcast.com/blog/changes-to-community-edition/

Reviewer: ```suggestion
In March 2024 we made some updates to the Community Edition of Hazelcast Platform, including making Hazelcast Platform Operator 5.13 closed source, which means its cluster creation capabilities will no longer be available in the Community Edition. 

You can find more information here: https://hazelcast.com/blog/changes-to-community-edition/.
```

Author: Makin repo closed-source and removing open source cluster creation capabilities are two seperate things so I think we should not use `which means` in here

Author: I have addressed other parts with https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/228/commits/39051d12189c8536137a6a807913ea8e399f2dc2

Reviewer: Ok, suggested another version

== WAN Replication Performance and Usability Enhancements

> For performant WAN Replication, all members should be connected directly by source clusters. To achieve this topology, we have intorudced `WithExposeExternally` parameter for Advanced WAN Network configuration at Hazelcast custom resource, see xref:advanced-networking.adoc#configuring-wan-protocol-sockets[Configuring WAN Protocol Sockets]

Reviewer: ```suggestion
For best performance using WAN Replication, all members should be connected directly by source clusters. To achieve this topology, we have introduced the `WithExposeExternally` parameter for Advanced WAN Network configuration. For more info, see xref:advanced-networking.adoc#configuring-wan-protocol-sockets[Configuring WAN Protocol Sockets].
```

Not sure what ' at Hazelcast custom resource' means here so suggestion includes taking it out...

Author: `Custom Resource` is kubernetes term, see https://kubernetes.io/docs/concepts/extend-kubernetes/operator/

We should keep it.

Author: I have addressed other parts with https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/228/commits/39051d12189c8536137a6a807913ea8e399f2dc2

Reviewer: It's maybe just the preposition that reads wrong to me i.e. "at". 

...we have introduced the `WithExposeExternally` parameter for configuring the Advanced WAN Network for the Hazelcast custom resource.

> In the 5.13 release, improvements have been made to the WANReplication custom resource to provide more accurate and informative status and validations.

Reviewer: ```suggestion
In the 5.13 release, improvements have been made to the WANReplication custom resource to provide more accurate and detailed statuses and validations.
```

== Multi-member routing for Java clients

With the release of Hazelcast Platform 5.5, you can use multi-member routing to provide greater performance and stability for Java client applications connecting to geographically dispersed clusters. 

> On the Platform Operator side, you can utilize this feature using 'High Availability Mode'. For more information, see xref:high-availability-mode.adoc##high-availability-mode-and-multi_member-routing[High Availability Mode and MULTI_MEMBER Routing] 

Reviewer: ```suggestion
On the Platform Operator side, you can utilize this feature using 'High Availability Mode'. For more information, see xref:high-availability-mode.adoc##high-availability-mode-and-multi_member-routing[High Availability Mode and MULTI_MEMBER Routing].
```

> On the Platform Operator side, you can utilize this feature using 'High Availability Mode'. For more information, see xref:high-availability-mode.adoc##high-availability-mode-and-multi_member-routing[High Availability Mode and MULTI_MEMBER Routing] 

Reviewer: ```suggestion
On the Platform Operator side, you can utilize this feature using High Availability Mode. For more info, see xref:high-availability-mode.adoc##high-availability-mode-and-multi_member-routing[High Availability Mode and MULTI_MEMBER Routing] 
```

== Additional Improvements

For detailed release notes that include new features and enhancements, breaking changes, deprecations and other fixes, see xref:release-notes.adoc[5.13 Release Notes].

To get started with Platform Operator, see xref:get-started.adoc[Deploy a Cluster with the Hazelcast Platform Operator for Kubernetes].

== API Reference

> Please refer to the xref:api-ref.adoc[API Reference]. for a complete list of available parameters for custom resources.

Reviewer: ```suggestion
Please refer to the xref:api-ref.adoc[API Reference] for a complete list of available parameters for custom resources.
```
